packages/flutter/lib/src/widgets/overflow_bar.dart

```diff
@@ -0,0 +1,482 @@
+  ///    overflows.
+  final OverflowBarAlignment overflowAlignment;
+
+  /// Defines the order that the [children] appear in, if
+  /// the horizontal layout overflows.
+  ///
+  /// This parameter is only used if the horizontal layout overflows, i.e.
+  /// if there isn't enough horizontal room for the [children] and [spacing].
+  ///
+  /// If the children do not fit into a single row, then they
+  /// are arranged in a column. The first child is at the top of the
+  /// column if this property is set to [VerticalDirection.down], since it
+  /// "starts" at the top and "ends" at the bottom. On the other hand,
+  /// the first child will be at the bottom of the column if this
+  /// property is set to [VerticalDirection.up], since it "starts" at the
+  /// bottom and "ends" at the top.
+  ///
+  /// Defaults to [VerticalDirection.down].
+  ///
+  /// See also:
+  ///
+  ///  * [overflowAlignment], which defines the horizontal alignment
+  ///    of the children within the vertical "overflow" layout.
+  final VerticalDirection overflowDirection;
+
+  /// Determines the order that the [children] appear in for the default
+  /// horizontal layout, and the interpretation of
+  /// [OverflowBarAlignment.start] and [OverflowBarAlignment.end] for
+  /// the vertical overflow layout.
+  ///
+  /// For the default horizontal layout, if [textDirection] is
+  /// [TextDirection.rtl] then the last child is laid out first. If
+  /// [textDirection] is [TextDirection.ltr] then the first child is
+  /// laid out first.
+  ///
+  /// See also:
+  ///
+  ///  * [overflowDirection], which defines the order that the
+  ///    [OverflowBar]'s children appear in, if the horizontal layout
+  ///    overflows.
+  final TextDirection textDirection;
+
+  /// {@macro flutter.widgets.Clip}
+  ///
+  /// Defaults to [Clip.none], and must not be null.
+  final Clip clipBehavior;
+
+  @override
+  _RenderOverflowBar createRenderObject(BuildContext context) {
+    return _RenderOverflowBar(
+      spacing: spacing,
+      overflowSpacing: overflowSpacing,
+      overflowAlignment: overflowAlignment,
+      overflowDirection: overflowDirection,
+      textDirection: textDirection ?? Directionality.of(context),
+      clipBehavior: clipBehavior,
+    );
+  }
+
+  @override
+  void updateRenderObject(BuildContext context, _RenderOverflowBar renderObject) {
+    renderObject
+      ..spacing = spacing
+      ..overflowSpacing = overflowSpacing
+      ..overflowAlignment = overflowAlignment
+      ..overflowDirection = overflowDirection
+      ..textDirection = textDirection ?? Directionality.of(context)
+      ..clipBehavior = clipBehavior;
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DoubleProperty('spacing', spacing, defaultValue: 0));
+    properties.add(DoubleProperty('overflowSpacing', overflowSpacing));
```

Comment:
To match the other lines
```suggestion
    properties.add(DoubleProperty('overflowSpacing', overflowSpacing, defaultValue: 0));
```

Reply:
👍